.storybook/preview.tsx

```diff
@@ -156,7 +156,6 @@ const preview: Preview = {
     },
   },
   parameters: {
-    actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
`argsTypesRegex` broke the SB migration several times and the only way to continue was in its removal. The [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#implicit-actions-can-not-be-used-during-rendering-for-example-in-the-play-function) gets into the thick of it, but it seems [the preferred way to target specific events](https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function) in each Story.